pkg/git/git_test.go

```diff
@@ -420,8 +439,17 @@ func (u *UtilsGitMock) plainClone(path string, isBare bool, o *git.CloneOptions)
 	return nil, nil
 }
 
+func (u *UtilsGitMock) plainOpen(path string) (*git.Repository, error) {
+	u.path = path
+	return nil, nil
```

Comment:
[minor] maybe in the future - when using that mock for opening a git repo in a broader way - it would be cool to return here some mocked repo (... which can be set on the mock in advance). But for now returning `nil` is basically fine ...

Reply:
on request.